src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
+@import './card';
+@import './stars';
```

Comment:
Consider using `@use` or `@forward` instead of `@import` as `@import` is deprecated in newer versions of Sass. This change can improve performance and maintainability.